tests/acceptance/pageObjects/FilesPageElement/filesList.js

```diff
@@ -366,6 +376,16 @@ module.exports = {
       this.api.execute('scrollTo(0,' + offset + ')')
       return this
     },
+    waitForFileRenameVisible: async function (fileName, elementType = 'any') {
+      const rowSelector = this.getFileRowSelectorByFileName(fileName, elementType)
+      let fileRename = null
+      await this.waitForFileVisible(fileName, (elementType = 'any'))
+      await this.getText('xpath', rowSelector, (result) => {
+        const res = result.value
+        fileRename = res.split('\n')
+      })
+      return fileRename[1]
```

Comment:
does the `result.value` always contains `\n` character?
please be cautious while doing `fileRename[1]`, the `fileRename` can still or null or it could be just a string, not an array.

Reply:
The result.value returns string 
```
Select file
lore.txt
699 B 10 seconds ago 
```
which is then converted into array 
` [ 'Select file', 'lore.txt', '699 B 10 seconds ago' ] `

Reply:
okay, please add some comments about the splitting